shared/src/business/entities/EntityConstants.js

```diff
@@ -1320,6 +1278,48 @@ const TRIAL_STATUS_TYPES = {
     new: true,
     label: 'Definite Trial',
   },
+  motionToDismiss: {
+    deprecated: false,
+    displayOrder: 9,
+    new: true,
+    label: 'Motion',
+  },
+  recall: {
+    deprecated: false,
+    displayOrder: 2,
+    label: 'Recall',
+  },
+  continued: {
+    deprecated: false,
+    displayOrder: 4,
```

Comment:
Does `displayOrder` get used? Since this story was concerning reordering the options, I was expecting the display order to maybe have changed for these items but maybe I'm misunderstanding

Reply:
The `displayOrder` defines the order in which these values are displayed in the list of filters, while the order of array defines the order in which they are displayed in the status dropdown

Reply:
Could probably be named better